### ISSUE TITLE
Build failed on riscv64

### ISSUE THREAD

**Issue by imguoguo** [mfem/mfem#3874]
When attempting to compile on a riscv64 machine, the following error message is displayed:

```
[233/420] /usr/bin/g++  -I/builddir/build/BUILD/mfem-4.5.2/riscv64-redhat-linux-gnu -I/builddir/build/BUILD/mfem-4.5.2 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -fasynchronous-unwind-tables -fstack-clash-protection -fno-omit-frame-pointer -DNDEBUG -std=c++11 -march=native -Wall -pedantic --param max-completely-peel-times=3 -MD -MT miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o -MF miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o.d -o miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o -c /builddir/build/BUILD/mfem-4.5.2/miniapps/performance/ex1.cpp
FAILED: miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o 
/usr/bin/g++  -I/builddir/build/BUILD/mfem-4.5.2/riscv64-redhat-linux-gnu -I/builddir/build/BUILD/mfem-4.5.2 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -fasynchronous-unwind-tables -fstack-clash-protection -fno-omit-frame-pointer -DNDEBUG -std=c++11 -march=native -Wall -pedantic --param max-completely-peel-times=3 -MD -MT miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o -MF miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o.d -o miniapps/performance/CMakeFiles/performance_ex1.dir/ex1.cpp.o -c /builddir/build/BUILD/mfem-4.5.2/miniapps/performance/ex1.cpp
g++: error: '-march=native': ISA string must begin with rv32 or rv64
g++: error: '-march=native': ISA string must begin with rv32 or rv64
```

It appears that the riscv64 architecture does not support the "march=native" option.

**Comment by tzanio** [comment]
Thanks for reporting this and proposing a fix, @imguoguo !

**Comment by tzanio** [comment]
Resolved on #3875
